.travis.yml

```diff
@@ -3,29 +3,24 @@ language: go
 go:
   - 1.9.x
 
-cache:
-  directories:
-    - vendor
-
 before_install:
-  - npm install -g ganache-cli
   - go get github.com/onsi/gomega
   - go get github.com/onsi/ginkgo/ginkgo
   - go get github.com/mattn/goveralls
   - go get github.com/golang/dep/cmd/dep
-  - go get github.com/golang/lint/golint
-  - go get github.com/haya14busa/goverage
+  - go get golang.org/x/lint/golint
+  - go get github.com/loongy/covermerge
   - dep ensure
 
 script:
-  - go vet ./...
-  - golint -set_exit_status `go list ./... | grep -Ev "(stackint/asm|vendor)"`
-  - go run cmd/testnetwork/main.go -sleep 10 &
-  - sleep 10
-  - goverage -v -coverprofile=coverprofile.out ./...
-
-after_success:
-  - sed -i '/rpc.pb.go/d' coverprofile.out
+  # - go vet ./...
+  # - golint -set_exit_status `go list ./... | grep -Ev "(stackint/asm|vendor)"`
+  # - golint `go list ./... | grep -Ev "(stackint/asm|vendor)"`
```

Comment:
`go vet` and `golint` can be re-enabled now

Reply:
Good idea, let's re-enable this once everything else is ready to go. Then we can go through and document types and functions in bulk.